Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang AS build
+
+COPY . /opt/conndetect
+WORKDIR /opt/conndetect
+RUN make
+
+FROM alpine
+
+RUN apk add --update gcompat && \
+    rm -rf /var/cache/apk
+COPY --from=build /opt/conndetect/out/conndetect /usr/bin/conndetect
+
+# ENTRYPOINT ["/usr/bin/conndetect"]
```

Comment:
Nitpick: we tend to discourage committing commented out sections unless we really think they'll get revived later on.

Reply:
oh, I actually intended to include that line, I commented it out for testing, thank you! :D